index.html

```diff
@@ -61,5 +61,276 @@ <h4 class="connect">LET'S CONNECT</h4>
+                    <div>
+                        <ul class="language-tags">
+                            <li class="html-lan">HTML</li>
+                            <li class="css">CSS</li>
+                            <li class="javascript">Javascript</li>
+
+                        </ul>
+                    </div>
+
+                    <div class="action">
+                        <button class="action-button">See project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="works">
+                <div class="snapshot">
+                    <img src="assets/images/multipoststories2.png"
+                        alt="snapshot" />
+                </div>
+                <div class="card-info">
+                    <div class="primary-text">
+                        <h2>Tonic</h2>
+                        <div class="work-info">
+                            <div class="client">
+                                <p>CANOPY</p>
+                            </div>
+                            <div class="counter"></div>
+                            <div class="role">
+                                <p>Backend Dev</p>
+                            </div>
+                            <div class="counterone"></div>
+
+                            <div class="year">
+                                <p>2015</p>
+                            </div>
+
+                        </div>
+                    </div>
+                    <div class="description-text">
+                        <p>A daily selection of privately personalized reads; no
+                            accounts or
+                            sign-ups required.</p>
+                    </div>
+                    <div>
+                        <ul class="language-tags">
+                            <li class="html-lan">HTML</li>
+                            <li class="css">CSS</li>
+                            <li class="javascript">Javascript</li>
+
+                        </ul>
+                    </div>
+
+                    <div class="action">
+                        <button class="action-button">See project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="about-container">
+            <section class="about-myself">
+                <div class="top">
+                    <h2>About Myself</h2>
+                    <p>Hello I’m a software developer! I can help you build a
+                        product ,
+                        feature or website. Look through some of my work and
+                        experience! If
+                        you like what you see and have a project you need coded,
+                        don’t
+                        hesitate to contact me.</p>
+
+                    <div class="socials">
+                        <div class="support-text">
+                            <p>Let's connect</p>
```

Comment:
- [ ] This is also a header that should use hx, you can consider using h3 or any downward.